### PR TITLE
fix and clean upload in admin

### PR DESCRIPTION
**Pull request by urirot**
- Fix fail to upload in admin 
- Change secrets to secrets_dict - the name causes an error when using python 3.8.10 (another secrets directory is used that overrides this one)
- Remove current_files level, so we only have latest copy stored for uploads

### CODE REVIEW

File: src/server/pipeline/clean_and_load_data.py [hunk 1]
```diff
@@ -16,7 +16,7 @@ def start(connection, pdp_contacts_df, file_path_list):
     manual_matches_df = pd.DataFrame()
     
     for uploaded_file in file_path_list:
-        file_path = os.path.join(CURRENT_SOURCE_FILES_PATH, uploaded_file)
+        file_path = os.path.join(RAW_DATA_PATH, uploaded_file)
```
Comment (c-simpson) [inline src/server/pipeline/clean_and_load_data.py:19]: In this code I'm seeing @ L28
`df = pd.read_csv((io.BytesIO(open(file_path, "rb").read())), encoding='iso-8859-1')`

that `file_path` is a directory, not a full path to a file.

Comment (urirot) [reply src/server/pipeline/clean_and_load_data.py:19]: Just checked that, it is actually still a file path. the CURRENT_SOURCE_FILE_PATH was also of a folder and not a file. the path.join adds the suffix to the path that directs it to the file in that directory

Comment (c-simpson) [reply src/server/pipeline/clean_and_load_data.py:19]: So I guess my real issue is that at line 19
RAW_DATA_PATH is '/app/static/raw_data/'
and 'uploaded_file' is `current`   so when you join them I get '/app/static/raw_data/current',  which is an existing directory, not a file.



image


I've uploaded **Salesforce_PAWS Donation_6_14_2021_TO_7_27_2021.xlsx** and am hitting 'Run Data Analysis'. Where should the file be?

Comment (urirot) [reply src/server/pipeline/clean_and_load_data.py:19]: behaves different on my end. I'm running locally that's why the path is not the docker one.
raw_data

and all files should be stored under raw_data/
make sure the docker was rebuilt etc? if not let's discuss next meeting

Comment (c-simpson) [reply src/server/pipeline/clean_and_load_data.py:19]: I was running locally (Windows). I deleted everything under raw_data and it ran fine. So it appears somemthing was getting confused. 
Not sure if it's worth having code clear those dirs on startup?

**Pull request merged**